### PR TITLE
translate docs on child cluster visiting

### PR DESCRIPTION
**Pull request by MentosL**
translate document，pls check it

### CODE REVIEW

**Comment by dixudx** [comment]
@MentosL Thanks for your contribution. Please squash your commits. Thanks.

**Comment by MentosL** [comment]
> @MentosL Thanks for your contribution. Please squash your commits. Thanks.

Okay, let's handle the party

**Comment by MentosL** [comment]
> @MentosL Thanks for your contribution. Please squash your commits. Thanks.

solved

**Review by dixudx — COMMENTED** [review]
Thanks for your contribution. Good job.

We just need a few updates.

File: content/zh-cn/docs/tutorials/cluster-management/visiting-child-clusters-with-rbac/construct-a-dedicated-kubeconfig.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
-Append `/apis/proxies.clusternet.io/v1alpha1/sockets/<CLUSTER-ID>/proxy/https/<SERVER-URL>`
-or `/apis/proxies.clusternet.io/v1alpha1/sockets/<CLUSTER-ID>/proxy/direct` at the end of original **parent cluster**
-server address
+在`/apis/proxies.clusternet.io/v1alpha1/sockets/<CLUSTER-ID>/proxy/https/<SERVER-URL>`
+或者`/apis/proxies.clusternet.io/v1alpha1/sockets/<CLUSTER-ID>/proxy/direct` 在**parent cluster**的末尾
+附加服务器地址
 
-- `CLUSTER-ID` is a UUID for your child cluster, which is auto-populated by `clusternet-agent`, such as dc91021d-2361-4f6d-a404-7c33b9e01118. You could get this UUID from objects `ClusterRegistrationRequest`,
-  `ManagedCluster`, etc. Also this UUID is labeled with key `clusters.clusternet.io/cluster-id`.
+- `CLUSTER-ID` 是您的子集群的 UUID，由 `clusternet-agent` 自动填充，例如 dc91021d-2361-4f6d-a404-7c33b9e01118。 您可以从对象`ClusterRegistrationRequest`中获取此 UUID，
```
[the remark this answers is not in the document]
Comment (MentosL) [reply content/zh-cn/docs/tutorials/cluster-management/visiting-child-clusters-with-rbac/construct-a-dedicated-kubeconfig.md]: solved

File: content/zh-cn/docs/tutorials/cluster-management/visiting-child-clusters-with-rbac/construct-a-dedicated-kubeconfig.md [hunk 2]
```diff
@@ -111,7 +111,7 @@ users:
     client-certificate-data: REDACTED
     client-key-data: REDACTED
 $
-$ # modify user part to below
+$ # 修改用户部分到下面
```
[the remark this answers is not in the document]
Comment (MentosL) [reply content/zh-cn/docs/tutorials/cluster-management/visiting-child-clusters-with-rbac/construct-a-dedicated-kubeconfig.md]: solved

File: content/zh-cn/docs/tutorials/cluster-management/visiting-child-clusters-with-rbac/construct-a-dedicated-kubeconfig.md [hunk 3]
```diff
@@ -162,7 +161,7 @@ users:
...
```
[the remark this answers is not in the document]
Comment (MentosL) [reply content/zh-cn/docs/tutorials/cluster-management/visiting-child-clusters-with-rbac/construct-a-dedicated-kubeconfig.md]: solved

**Comment by dixudx** [comment]
Thanks for your contribution. Nice work! @MentosL

**Pull request merged**